de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5462240_sys_gh1513-translating_en_us.sql

```diff
@@ -0,0 +1,80 @@
+update ad_field_trl set name = 'Delivery method' ,istranslated = 'Y' where ad_field_id = 2877 and ad_language = 'en_US';
```

Comment:
@teosarca totally do agree. 
can we move that problem to another issue as its hard to solve when translating?
at the moment its not translated at all although its used and documented.
so I am willing to take the risk that we lose the translation of a field IF the element is changed.